app/auth/routes.py

```diff
@@ -0,0 +1,95 @@
+"""
+Authentication settings for the BucketList API.
+
+Password verification and user registration takes place here.
+"""
+from flask import g, jsonify, request
+from flask_httpauth import HTTPBasicAuth
+
+from . import authe
+from app import errors
+from app.decorators import json
+from app.models import User
+
+auth = HTTPBasicAuth()
+
+
+@auth.verify_password
+def verify_password(token, password):
+    """
+    Verify token.
+
+    Verify token, password doesn't need to be present here. The token is
+    going to be in the request headers always.
+    """
+    user = User.verify_auth_token(token)
+
+    if not user:
+        return False
+
+    g.user = user
+    return True
+
+
+@authe.route('/login', methods=['POST'])
+def login():
+    """
+    Verify a user's identity.
+
+    Verify the user's identity and returns a token.
```

Comment:
I have added aa better description to the docstring body.

So i was going through the documentation and turns out both styles of writing multi-line docstrings are equivalent. I usually start my doc string title on a new line because sometimes it exceeds 79 characters. So having the triple-quotes on a separate line makes room for 3 additional characters.

<img width="855" alt="screen shot 2017-04-06 at 7 26 04 am" src="https://cloud.githubusercontent.com/assets/25608335/24740700/740b7b3a-1a9a-11e7-86de-b15ff44a280d.png">